src/index.ts

```diff
@@ -1 +1,34 @@
-//Define class here
\ No newline at end of file
+//Define class here
+
+export interface People {
+  readonly name: string;
+  readonly age: number;
+}
+
+export const copyPeopleArray = (people: ReadonlyArray<People>) => {
```

Comment:
not ret required here because technically we aren't starting our typescript study, but since you're typing the input here, we should also be typing the output, aka, the function's return value
```suggestion
export const copyPeopleArray = (people: ReadonlyArray<People>): ReadonlyArray<People> => {
```